source/extensions/filters/network/dubbo_proxy/hessian_utils.cc

```diff
@@ -516,6 +558,12 @@ std::string HessianUtils::readByte(Buffer::Instance& buffer) {
   return result;
 }
 
+void HessianUtils::writeString(Buffer::Instance& buffer, const std::string& str) {
```

Comment:
replace `const std::string& str` to `const absl::string_view&` ?

Reply:
done